batch7crud-roll7/src/main/java/com/lftechnology/batch7crud/controller/StudentListController.java

```diff
@@ -0,0 +1,173 @@
+    String name = request.getParameter("name");
+    String address = request.getParameter("address");
+    String roll = request.getParameter("roll");
+    try {
+
+      int rollNum = Integer.parseInt(roll);
+      Student student = new Student();
+      student.setName(name);
+      student.setAddress(address);
+      student.setRoll(rollNum);
+
+      StudentServices studentService = new StudentServices();
+      studentService.addNew(student);
+      response.sendRedirect(request.getContextPath() + "/" + CommonConstants.LIST_URL);
+    } catch (NumberFormatException e) {
+      request.setAttribute("error", "invalid roll");
+      request.getServletContext().getRequestDispatcher(CommonConstants.NEW_ENTRY_VIEW).forward(request, response);
+    }
+
+  }
+
+  private void edit(HttpServletRequest request, HttpServletResponse response, int id) throws ServletException, IOException, DataException {
+    StudentServices studentService = new StudentServices();
+
+    request.setAttribute("student", studentService.fetchById(id));
+    request.getRequestDispatcher(CommonConstants.EDIT_ENTRY_VIEW).forward(request, response);
+
+  }
+
+  private void editProcess(HttpServletRequest request, HttpServletResponse response, int id)
+    throws ServletException, IOException, DataException {
+    String name = request.getParameter("name");
+    String address = request.getParameter("address");
+    int roll = Integer.parseInt(request.getParameter("roll"));
+    Student student = new Student();
+    student.setName(name);
+    student.setAddress(address);
+    student.setRoll(roll);
+    student.setId(id);
+
+    StudentServices studentService = new StudentServices();
+    studentService.update(student);
+    response.sendRedirect(request.getContextPath() + File.separator + CommonConstants.LIST_URL);
+
+  }
+
+  private void deleteProcess(HttpServletRequest request, HttpServletResponse response, int id)
+    throws ServletException, IOException, DataException {
+    StudentServices studentService = new StudentServices();
+    studentService.delete(id);
+    response.sendRedirect(request.getContextPath() + File.separator + CommonConstants.LIST_URL);
+
+  }
+
+  private void show(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException, DataException {
+    try {
+      int id = urlInteger(request, 2);
+      StudentServices studentService = new StudentServices();
+      Student student = studentService.fetchById(id);
+      if (student == null)
+        showErrorPage(request, response);
+      else {
+        request.setAttribute("student", student);
+        request.getRequestDispatcher(CommonConstants.SHOW_STUDENT_VIEW).forward(request, response);
+      }
+    } catch (NumberFormatException e) {
+      showErrorPage(request, response);
+    }
+  }
+
+//  public void showErrorPage(HttpServletRequest request, HttpServletResponse response) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)